File: patterns/c-patterns/pattern17.c
// Pattern 17: ...

// ****1
// ***21
// **321
// *4321
// 54321

#include <stdio.h>
int main()
{
    int i, j;
    int n = 5;
    for (i = 1; i <= n; i++)
    {
        for (j = n; j >= 1; j--)
        {
            if (j > i)
                printf("*");
            else
                printf("%d", j);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern17.js
// Pattern 17: ...

// ****1
// ***21
// **321
// *4321
// 54321

const n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = n; j >= 1; j--) {
        if (j > i) {
            row += '*';
        } else {
            row += j;
        }
    }

    console.log(row);
}
